php/thankYou.php

```diff
@@ -350,8 +350,16 @@
                     }
 
                 //About you
-
-                $otherHeardAbout = (!empty($otherHowDidHear)) ? $howDidHear = $otherHowDidHear : '';
+                //if ($howDidHear === 'other') {
+                if (!empty($otherHowDidHear)) {
+                    echo "<h1>TEST 1 $howDidHear $otherHowDidHear</h1>";
+                }
+                else {
+                    echo "<h1>FAIL</h1>";
+                }
```

Comment:
Not sure if this if statement is needed? If it is, remove the "TEST 1". Also, it always prints "FAIL" even if you choose a correct option in the dropdown.